### PR TITLE
Updated recipe/meta.yaml to version 17.4.0

### PR DESCRIPTION
**Pull request by jochym**

### CODE REVIEW

**Comment by jochym** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = "attrs" %}
-{% set version = "17.3.0" %}
-{% set sha256 = "c78f53e32d7cf36d8597c8a2c7e3c0ad210f97b9509e152e4c37fa80869f823c" %}
+{% set version = "17.4.0" %}
+{% set sha256 = "1c7960ccfd6a005cd9f7ba884e6316b5e430a3f1a6c37c5f87d8b43f83b54ec9" %}
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: Please drop build number back to 0.

Comment (jochym) [reply recipe/meta.yaml]: Thanks! Missed that.

**Comment by nicoddemus** [comment]
Thanks @jochym!

**Pull request merged**